src/vmware/HISTORY.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 6.0.1 (2023-09)
+- Update to AVS 2023-03-01 API
+- Add `az vmware private-cloud add-extended-network-blocks`
+- Add `az vmware private-cloud delete-extended-network-blocks`
+- Updated `az vmware private-cloud create` with new `ext_nw_blocks` parameter
```

Comment:
Is spelling it out with `extended_network_blocks` to long? Does something not work?

Reply:
Yea there's a linter error that parameter name is too long